core/src/test/java/org/hibernate/ogm/backendtck/elementcollection/DuplicatesAndOrderOfAssociationItemsTest.java

```diff
@@ -0,0 +1,152 @@
+/*
+ * Hibernate OGM, Domain model persistence for NoSQL datastores
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.ogm.backendtck.elementcollection;
+
+import static org.fest.assertions.Assertions.assertThat;
+
+import java.util.List;
+
+import org.hibernate.ogm.utils.OgmTestCase;
+import org.hibernate.ogm.utils.TestForIssue;
+
+import org.junit.After;
+import org.junit.Test;
+
+/**
+ * When a {@link javax.persistence.OrderColumn} is defined on association mapped as {@link List},
+ * order and duplicates will be preserved over different loads.
+ *
+ * Otherwise that will not guarantee. It will depend on the specific datastore behavior.
```

Comment:
Can we keep this javadoc simpler?
Something like:
{quote}
Tests for associations and collections of elements containing duplicates, with or without @OrderColumn.
{quote}

Reply:
Of course we can